.github/workflows/nodejs.yml

```diff
@@ -52,13 +52,12 @@ jobs:
         uses: wagoid/commitlint-github-action@v1
 
   test:
-    name: Test - ${{ matrix.os }} - Node v${{ matrix.node-version }}, Webpack ${{ matrix.webpack-version }}
+    name: Test - ${{ matrix.os }} - Node v${{ matrix.node-version }}
 
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
         node-version: [10.x, 12.x, 14.x]
-        webpack-version: [latest, next]
```

Comment:
Don't remove this, we should test it on webpack@4 and webpack@5

Reply:
Currently, the `next` is tagged to `5.0.0-rc.6`. I'll add `webpack-4`.

Reply:
Change it on:
```
webpack-version: [4, latest]
```

Reply:
Done